src/hyp3_isce2/metadata/templates/insar_burst/insar_burst_readme.md.txt.j2

```diff
@@ -65,19 +300,17 @@ The basic steps in Sentinel-1 Burst InSAR processing are as follows:
 6. Set the Configure file for the InSAR process
 
 *InSAR Processing*
```

Comment:
This section doesn't call out the step where bursts are merged. Is that something we should include?

Reply:
That's a question for @forrestfwilliams